network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -132,7 +132,7 @@ def exec(self):
                            full_explanation, res)
 
     def compute_query_output(self, query_answer):
-        return self.get_query_output(query_answer, query_answer.bool_result)
+        return self.get_query_output(query_answer, add_explanation=query_answer.bool_result)
```

Comment:
before this change the query output always included only one part of the (output_result , output_explanation) , this ensures if the two parts exist in the queryAnswer so both will be printed in the query output message.
An example test: query name : with_emptiness , test file : tests/calico_testcases/example_policies/testcase19-profiles/testcase19-scheme.yaml

output before:

> Running query with_emptiness
Ingress rule no. 1 in NetworkPolicy np9-0-cnc-fe-between-namespaces-with-deny-all-profiles/cnc-fe/testcase19-9-1-specific-ingress-with-policies-based-apply-labels selects no source pods
NetworkPolicy np11-global-without-label-to-apply/testcase19-11-global-open-within-cnc-kt-with-labels does not select any pods
Ingress rule no. 1 in NetworkPolicy np11-global-without-label-to-apply/testcase19-11-global-open-within-cnc-kt-with-labels selects no destination pods
Egress rule no. 1 in NetworkPolicy np11-global-without-label-to-apply/testcase19-11-global-open-within-cnc-kt-with-labels selects no source pods

output after:

> Running query with_emptiness
There are empty NetworkPolicies and/or empty ingress/egress rules in np9-0-cnc-fe-between-namespaces-with-deny-all-profiles
Ingress rule no. 1 in NetworkPolicy np9-0-cnc-fe-between-namespaces-with-deny-all-profiles/cnc-fe/testcase19-9-1-specific-ingress-with-policies-based-apply-labels selects no source pods
There are empty NetworkPolicies and/or empty ingress/egress rules in np11-global-without-label-to-apply
NetworkPolicy np11-global-without-label-to-apply/testcase19-11-global-open-within-cnc-kt-with-labels does not select any pods
Ingress rule no. 1 in NetworkPolicy np11-global-without-label-to-apply/testcase19-11-global-open-within-cnc-kt-with-labels selects no destination pods
Egress rule no. 1 in NetworkPolicy np11-global-without-label-to-apply/testcase19-11-global-open-within-cnc-kt-with-labels selects no source pods